client-src/elements/chromedash-guide-new-page_test.ts

```diff
@@ -54,7 +54,7 @@ describe('chromedash-guide-new-page', () => {
     assert.include(featureTypeFormField.outerHTML, 'Feature removal');
 
     // submit button exists
-    const submitButton = component.shadowRoot.querySelector(
+    const submitButton = component.renderRoot.querySelector(
       'input[type="submit"]'
     );
```

Comment:
Is there a reason that `!` is needed in the previous querySelector invocations, but not the one for the submit button?

Reply:
it was bc we inserted this to exist so the editor knows it's not null while the previous ones do not have such insertions